cmd/root.go

```diff
@@ -120,46 +120,48 @@ func bindEnv(flags ...string) bindFunc {
 	}
 }
 
+type functionOverride func (root string) error
+
+type strPropSetter func (f *faas.Function, val string)
+
+func createGenericOverride(setter strPropSetter) func (override string) functionOverride {
```

Comment:
This is kinda funny. Maybe we could just introduce some new `overrideParam`  struct and make it parameter of `functionWithOverrides`.

Reply:
> This is kinda funny. Maybe we could just introduce some new `overrideParam` struct and make it parameter of `functionWithOverrides`.

That would probably be a lot more readable.